src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -130,7 +132,13 @@ export const JiraConfigurationGet = async (
 			isGitHubCloudApp: await isGitHubCloudApp(gitHubAppId)
 		});
 
-		req.log.debug("Jira configuration rendered successfully.");
+		sendAnalytics(AnalyticsEventTypesEnum.ScreenEvent, {
+			name: AnalyticsScreenEventsEnum.GitHubConfigScreenEventName,
+			jiraHost,
```

Comment:
I already chatter with Josh about this and he has said he's going to revert the hashing changes on the jiraHost because it makes it impossible to look through the logs.

Reply:
lol, all good then.